consensus/istanbul/core/commit.go

```diff
@@ -66,6 +67,11 @@ func (c *core) handleCommit(msg *message, src istanbul.Validator) error {
 		return err
 	}
 
+	err = blscrypto.IsValidSignature(msg.CommittedSeal)
```

Comment:
Is the signer of the bls signature verified to be the sender of the message here?

Reply:
It's not. This was actually added pretty recently, to prevent invalid signatures causing problems later on.

I think the original logic didn't verify the CommittedSeal as well until much later - in the verifyHeader. Is that true? If yes, what's the thought behind it?